Extensions/timestamps.js

```diff
@@ -16,46 +16,47 @@ XKit.extensions.timestamps = new Object({
 	slow: true,
 
 	preferences: {
-		only_on_hover: {
-			text: "Only show timestamps when I hover over a post",
-			default: false,
-			value: false
-		},
-		only_relative: {
-			text: "Only show relative time (eg: 5 minutes ago)",
-			default: false,
-			value: false
-		},
-		only_inbox: {
-			text: "Only show timestamps on asks in my inbox",
-			default: false,
-			value: false
-		},
-
-		title_reblogs: {
-			text: "Reblog timestamps",
-			type: "separator"
-		},
-		do_reblogs: {
-			text: "Show timestamps on reblog comments",
+		inbox: {
+			text: "Show timestamps in the inbox",
 			default: true,
 			value: true
 		},
-		only_original: {
-			text: "Only put timestamps on original comments",
+		posts: {
+			text: "Show timestamps on posts",
 			default: true,
 			value: true
 		},
+		reblogs: {
+			text: "Reblog timestamps",
```

Comment:
okay okay one more suggestion—maybe this should say "Show timestamps on reblogs" just for parallelism? so it's a little more intuitive that this is a third independent options?

Reply:
i think the wording of the option values make it clear enough
also sorry i merged this while you were typing this! im very fast you see